pkg/operator/ceph/spec/spec.go

```diff
@@ -99,32 +100,39 @@ func PodVolumes(dataDirHostPath, namespace string, confGeneratedInPod bool) []v1
 	if confGeneratedInPod {
 		configVolume, _ = confGeneratedInPodVolumeAndMount()
 	}
-	return []v1.Volume{
+
+	v := []v1.Volume{
 		{Name: k8sutil.DataDirVolume, VolumeSource: dataDirSource},
 		configVolume,
-		StoredLogVolume(path.Join(dataDirHostPath, namespace, "log")),
```

Comment:
Instead of mounting both the `log` and `crash` dirs, what if we just mount the parent directory? I guess either way is fine though.